### PR TITLE
Dimension mode selection

### PR DESCRIPTION
**Pull request by kristofferlind**

### CODE REVIEW

File: src/qsSimpleKPI.less [hunk 1]
```diff
@@ -1,138 +1,225 @@
...
+  @import "../semantic/src/definitions/elements/icon.less";
+  // @import "../semantic/src/definitions/elements/label.less";
+  @import "../semantic/src/definitions/views/statistic.less";
+  @import "../semantic/src/definitions/elements/segment.less";
+
+  .ui.segments, .ui.cards {
+    overflow: auto;
+
+    .segment, .card {
+      min-width: auto !important;
```
Comment (ahmed-Bazzara) [inline src/qsSimpleKPI.less]: it should be "min-width : fit-content" in order for it to work for card and segment view

File: src/qsSimpleKPI.less [hunk 2]
```diff
@@ -1,138 +1,228 @@
...
+}
+
+
+.qv-object-qsstatistic {
+  width: 100%;
+  height: 100%;
+}
+.qv-object-qsstatistic {
+  @import "../semantic/src/definitions/views/card.less";
+  //@import "../semantic/src/definitions/elements/button.less";
```
Comment (kristofferlind) [inline src/qsSimpleKPI.less]: Considering the amount of changes in this file we might aswell clean up the commented includes aswell.

File: src/qsSimpleKPI.less [hunk 3]
```diff
@@ -1,138 +1,228 @@
...
+  @import "../semantic/src/definitions/elements/icon.less";
+  // @import "../semantic/src/definitions/elements/label.less";
+  @import "../semantic/src/definitions/views/statistic.less";
+  @import "../semantic/src/definitions/elements/segment.less";
+
+  .ui.segments, .ui.cards {
```
Comment (kristofferlind) [inline src/qsSimpleKPI.less]: There's an identical selector at the bottom and it would be preferable to merge them.

File: src/qsSimpleKPI.less [hunk 4]
```diff
@@ -1,138 +1,228 @@
...
+    padding: 1em;
+  }
+
+  .ui.icon.button>.icon, .ui.icon.buttons .button>.icon {
+    opacity: .9;
+    margin: 0;
+    vertical-align: top;
+  }
+
+  .ui.segments, .ui.cards{
```
Comment (kristofferlind) [inline src/qsSimpleKPI.less]: Really picky, but there's a space missing before { to keep it consistent (this line and a couple selectors below). Should possibly add a csslint aswell (new issue in that case)

**Pull request merged**